build/.vsts-ci.yml

```diff
@@ -966,3 +966,25 @@ jobs:
       cd samples/dockerbuilds/MIPS32
       docker build -t mipsiotbuild:latest . --network=host
     displayName: 'Build MIPS32'
+- job: crosscompile_arm
+  pool:
+    vmImage: 'ubuntu-20.04'
+  displayName: "Cross Compile (ARM)"
+  steps:
+  - script: |
+      sudo apt-get update && sudo apt-get install -y \
+      curl \
+      git \
+      python-software-properties \
+      build-essential \
+      pkg-config \
+      docker.io
+      sudo curl -sL https://deb.nodesource.com/setup_16.x | bash -
+      sudo apt-get install -y nodejs
+    displayName: 'Setup'
+  - script: |
+      cd samples/dockerbuilds/ARM
+      docker build -t armiotbuild:latest . --network=host
+      cd samples/dockerbuilds
```

Comment:
/home/vsts/work/_temp/bee4a43d-eb79-4907-8231-a303ef3ed034.sh: line 3: cd: samples/dockerbuilds: No such file or directory

look at two lines above. you are currently at samples/dockerbuilds/ARM